k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -2666,8 +2666,6 @@ spec:
                       type: boolean
                     prefilterDevice:
                       type: string
-                    prometheusServeAddr:
-                      type: string
```

Comment:
Do we allow removals like this?

Reply:
That is indeed the question. At some point we do because it is just confusing for end users if we leave it in. And no one can be using that since it doesn't do anything.

If we have a more clear deprecation guide, I can do the removal in a second PR.

Reply:
My understanding is that we need to leave them in unused. When we create a new API version, then we can remove it from the new version. My understanding might be inaccurate and I don't know how we're tracking items to change in the next API revision.

Reply:
I removed this bit now. Can be fixed at a later time, I guess